Tests/ReactiveSwiftTests/SignalProducerLiftingSpec.swift

```diff
@@ -1666,5 +1666,47 @@ class SignalProducerLiftingSpec: QuickSpec {
 				expect(latestValues?.1) == 2
 			}
 		}
+		
+		describe("groupBy") {
```

Comment:
It would be great if there are more test cases. At least this single test cases apparently did not catch the undisposed upstreams, caused by [this line](https://github.com/ReactiveCocoa/ReactiveSwift/pull/200/files#diff-ceb9f0d4c919bbf25ca4e3c8b7036b8bR1897).